lib/OnyxCache.js

```diff
@@ -36,7 +36,7 @@ class OnyxCache {
          * Captured pending tasks for already running storage methods
          * @type {Record<string, Promise>}
          */
-        this.pendingPromises = {};
+        this.pendingPromises = new Map();
```

Comment:
Could you update the docs above and leave a comment why we chose to use Map?

Reply:
What do you mean with the docs though? Docs are only generated from onyx.js, and not from other internal implementation files such as this one

Reply:
`@type {Record<string, Promise>}` I meant this